specification/eventgrid/data-plane/Microsoft.Web/stable/2018-01-01/Web.json

```diff
@@ -0,0 +1,679 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure App Service events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure App Service events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "WebAppDeletedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Web.AppDeleted event.",
+      "type": "object",
+      "properties": {
+        "appEventTypeDetail": {
+          "description": "Detail of action on the app",
```

Comment:
Just for awareness, you are creating an anonymous class here (a class with "action"). SDK will pick a random name based on the attribute usage (here I would guess something like "WebAppDeletedEventDataAppEventTypeDetail"). You might want to make this name clearer, you can add a `x-ms-client-name` attribute here to specify that name.